packages/bitcore-client/bin/wallet-send

```diff
@@ -50,11 +50,39 @@ const main = async () => {
       from: lastAddress,
       token
     };
-    const tx = await wallet.newTx(params);
+    let tx;
+    const txTemp = await wallet.newTx(params);
+    let utxosValueSum = 0;
+    let inputCount = 0;
+    let outputAmount = recipients[0].amount;
+    if (['BTC', 'BCH', 'DOGE', 'LTC'].includes(chain)) {
+      for (const utxo of utxos) {
+        if (utxosValueSum < outputAmount) {
+          utxosValueSum = utxosValueSum + utxo.value;
+          inputCount += 1;
+        }
+      }
+      if (utxosValueSum < outputAmount) {
+        console.error('wallet input was not sufficient to cover the output amount');
+        process.exit(1);
+      } else {
+        const estimatedBytes = txTemp.length / 2 + inputCount * (107 + 1);
```

Comment:
For my own edification, where is the 107 + 1 coming from?

Reply:
it's bytes per input signature plus wiggle room byte. check bitpay/lib/opsUtils/bitcoinCash.js